conformance/tests/typeddicts_operations.py

```diff
@@ -32,19 +32,33 @@ class Movie(TypedDict):
 movie = {"name": "", "year": 1900, "other": 2}  # E: extra key
 
 
-def func1(variable_key: str):
+def func1(variable_key: str, existing_movie: Movie):
     # > A key that is not a literal should generally be rejected.
     movie: Movie = {variable_key: "", "year": 1900}  # E: variable key
 
+    # Destructive operations.
+    existing_movie[variable_key] = 1982  # E: variable key
```

Comment:
I'd like to get clarification on the wording in the spec before we add these tests. I've started a discussion [here](https://discuss.python.org/t/indexing-typeddict-with-a-non-literal-variable/88929). Feel free to add your viewpoint to the thread.